### PR TITLE
`FilePickerApp` design updates, part 1: "Chrome"/layout structure

### PR DESCRIPTION
**Pull request by lyzadanger**
This is the first of 2 or 3 PRs to address some design technical debt in the `FilePickerApp`. The changes in this PR address the broader layout and design _structure_. A subsequent PR will address the file-picker buttons, which are, let's be honest, a little bit hideous.

This PR:

* Adds a grey backdrop to the app so that file-picker controls don't look at sea on larger screens (e.g. in BlackBoard, D2L...)
* Uses a `Card` and other common components to make design patterns more conventional
* Ensures that over-height content (but not headers or buttons) will scroll. Having said that, the full set of file-picker buttons _should_ fit in the shortest known context, the little Canvas modal, without scrolling.
* Drops in the shared `Select` component for the group-configuration step and styles adjacent controls. Note also that the "Continue" button will no longer move downward when the group-configuration control is enabled (by selecting the checkbox).

Yes, those buttons are still terrible! That's the next step.

This PR does not make any functional changes.

### Before

image

Group configuration screen:

image

In BlackBoard:

image

### After

image

Group configuration screen:

image

image

In BlackBoard:

image

Overheight content will scroll:

image

### CODE REVIEW

**Review by acelaya — APPROVED** [review]
Nice!

**Comment by lyzadanger** [comment]
Note: Slack conversation this morning with product folks and we are OK'd to move forward with these layout changes.

File: lms/static/scripts/frontend_apps/components/FilePickerApp.tsx [hunk 1]
```diff
@@ -171,47 +175,96 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
...
+    <main
+      className={classnames(
+        // Full-width and -height light-grey background
+        'bg-grey-1 w-full h-full p-2'
+      )}
+    >
+      {/**
+       * The <form> is styled as a constraining container that determines
+       * the Card's dimensions
+       */}
```
Comment (robertknight) [inline lms/static/scripts/frontend_apps/components/FilePickerApp.tsx:187]: This can just be a regular `/* ... */` non-JSDoc comment.

Comment (lyzadanger) [reply lms/static/scripts/frontend_apps/components/FilePickerApp.tsx:187]: Ah, yes, good ol' muscle memory.

File: lms/templates/file_picker.html.jinja2 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% extends "lms:templates/base.html.jinja2" %}
 
 {% block content %}
-  <div id="app"></div>
+  <div style="width: 100%; height: 100%;" id="app"></div>
```
[the remark this answers is not in the document]
Comment (lyzadanger) [reply lms/templates/file_picker.html.jinja2:4]: Yes, you're two steps ahead of me. I took me most of yesterday afternoon to get this work put together, and I didn't want to bloat scope, but this was already on my mind for follow-up (jinja consolidation).

**Pull request merged**